### ISSUE TITLE
Device Tree support in the Kernel

### ISSUE THREAD

**Issue by jkent** [ARMWorks/NanoPi-tasks#3]
Our Kernel currently uses platform data and not Device Tree for configuration.  This should be a fairly easy task to convert the platform data structures to their Device Tree equivalents.  This issue is partly blocked by #2.  Some sort of work in parallel will be needed for testing.

**Comment by jkent** [comment]
Won't fix, NanoPi is EOL
